src/asmjit/arm/armfeatures.cpp

```diff
@@ -329,11 +336,139 @@ ASMJIT_FAVOR_SIZE void detectCpu(CpuInfo& cpu) noexcept {
 }
 #endif // __linux__
 
+#if defined(__APPLE__)
+
+ASMJIT_FAVOR_SIZE void detectCpu(CpuInfo& cpu) noexcept {
+
+  // use sysctlbyname() to obtain CPU family; match on it as there is no good way to detect individual features
+  uint32_t cpuFamily = 0;
+  size_t l = sizeof(cpuFamily);
+  int res;
+  if ((res = sysctlbyname("hw.cpufamily", &cpuFamily, &l, NULL, 0)) != 0) {
```

Comment:
We have to handle this differently - if we fail to get the cpu family we should assume baseline features only, but not `exit(1)`

Reply:
Resolved for unknown arches, not yet resolved for failed sysctl.